src/shared/components/reactPlotlyWrapper/ReactPlotlyWrapper.tsx

```diff
@@ -0,0 +1,120 @@
+import React, {ReactElement} from 'react';
+import {observer} from "mobx-react";
+import {observable, runInAction} from "mobx";
+import Plotly from 'plotly.js';
+import classNames from 'classnames';
+
+export interface IReactPlotlyWrapperProps {
+    layout:any;
+    data:any; buildTooltip:(tooltipModel:any)=>JSX.Element
+}
+
+@observer
+export default class ReactPlotlyWrapper extends React.Component<IReactPlotlyWrapperProps, {}>
+{
+
+    private wrapper:HTMLDivElement;
```

Comment:
based on use this seems like it should be typed as a jquery object

Reply:
Mine works like this too, no need to use jQuery.

Reply:
further down we're using `wrapper.on`, on is a jQuery function isnt it